src/rest-server/src/models/job.js

```diff
@@ -395,7 +453,15 @@ class Job {
         appCompletedTime: frameworkStatus.applicationCompletedTimestamp,
         appExitCode: frameworkStatus.applicationExitCode,
         appExitDiagnostics: frameworkStatus.applicationExitDiagnostics,
```

Comment:
>appExitDiagnostics [](start = 8, length = 18)

If you can successfully find contaierStderr and runtimeError in applicationExitDiagnostics, please also delete them from applicationExitDiagnostics.

So that appExitDiagnostics will not duplicated with contaierStderr and runtimeError. Thus may confuse user.

Reply:
This explains: 
"launcher: = frameworkStatus.applicationExitDiagnostics - contaierStderr - runtimeError
"

---
In reply to: [276914338](https://github.com/Microsoft/pai/pull/2550#discussion_r276914338) [](ancestors = 276914338)